AnnoyancesFilter/sections/cookies_general.txt

```diff
@@ -893,6 +893,7 @@
 ###viewlet-cookiepolicy
 ###vscookieAlertCont
 ###vvc_cookie
+###wa_cmpOverlay
```

Comment:
Please remove this rule and add domain vandale.nl to the existing rule in cookies_specific

Reply:
Sure, done. @Sergey-Lyapin